tests/JavaScriptTest.js

```diff
@@ -240,7 +252,7 @@ function test64bit() {
 
   Stat.startStat(fbb);
   // 2541551405100253985 = 0x87654321(low part) + 0x23456789 * 0x100000000(high part);
-  Stat.addVal(fbb, new flatbuffers.Long(0x87654321, 0x23456789));    // the low part is Uint32
+  Stat.addVal(fbb, BigInt.asUintN(64, BigInt(0x87654321) + (BigInt(0x23456789) << BigInt(32)))); // the low part is Uint32
```

Comment:
There's no bigint constant syntax to put this all into 1 number?

Reply:
Should be, BigInt accepts hex notation in string form as input.

Reply:
My goal was to prove it works exactly the same by changing the unit tests as little as possible. Guess it's still a good test to pass a hex string or a BigInt literal as is done for asserting the getter value on L276